### PR TITLE
A simple friendly Quarg hail.

### PR DESCRIPTION
**Pull request by Lineth**
Not to steal pointedstick's thunder or anything. :3

### CODE REVIEW

**Comment by KiLEdEnNis** [comment]
what does geis mean? google translator won't tell
& "to set an example to all" not sure, is it "to set an example for all"?

**Comment by Bladewood** [comment]
https://en.wikipedia.org/wiki/Geis

File: data/hails.txt [hunk 1]
```diff
@@ -2479,6 +2479,36 @@ phrase "hostile bounty"
...
+		"those who live"
+	word
+		" "
+	word
+		"as we are sworn to do."
+		"as the Drak guide us."
+		"to uphold the sanctity of life."
+		"as is our geis."
+		"to set an example to all."
 # From the "Southern Television broadcast interruption."
```
Comment (Amazinite) [inline data/hails.txt]: Put some space between the bottom of the new hails and these old ones.

**Comment by Pointedstick** [comment]
"Geis" may be too obscure. How about "duty" instead?

File: data/hails.txt [hunk 2]
```diff
@@ -2479,6 +2479,38 @@ phrase "hostile bounty"
...
+		"the galaxy's children"
+		"those who live"
+	word
+		" "
+	word
+		"as we are sworn to do."
+		"as the Drak guide us."
+		"to uphold the sanctity of life."
+		"as is our geis."
+		"to set an example to all."
```
Comment (Pointedstick) [inline data/hails.txt]: for all, not too all

File: data/hails.txt [hunk 3]
```diff
@@ -2479,6 +2479,38 @@ phrase "hostile bounty"
...
+		"the young races"
+		"the galaxy's children"
+		"those who live"
+	word
+		" "
+	word
+		"as we are sworn to do."
+		"as the Drak guide us."
+		"to uphold the sanctity of life."
+		"as is our geis."
```
Comment (Pointedstick) [inline data/hails.txt]: "Geis" may be too obscure; how about "duty" instead?

File: data/hails.txt [hunk 4]
```diff
@@ -2479,6 +2479,38 @@ phrase "hostile bounty"
...
+	word
+		" "
+	word
+		"the galaxy"
+		"the young and foolish"
+		"the young races"
+		"the galaxy's children"
+		"those who live"
+	word
+		" "
```
Comment (Pointedstick) [inline data/hails.txt]: Let's add a comma.

File: data/hails.txt (hunk 3 above)
Comment (Lineth) [inline data/hails.txt]: I'm of the opinion that geis fits the Quarg's rather antiquated speech patterns better.

Comment (jafdy) [inline data/hails.txt]: Yes but that doesn't help if no one knows what it means.

Comment (Bladewood) [inline data/hails.txt]: It is an incredibly obscure word.
I couldn't find any internet dictionary that contained it.
I could only find it on Wikipedia and Wiktionary.
I do believe we should use a different word.

At least stuff like "sanctity" can be figured out from their roots, with some effort; "geis" is basically impossible to decipher.

Comment (Quantumshark) [inline data/hails.txt]: I think it's spelt "Geas".

Comment (MCOfficer) [inline data/hails.txt]: it's both possible, see https://en.wiktionary.org/wiki/geas#Irish

**Comment by endless-sky** [comment]
I kind of prefer the idea that the Quarg are just not a particularly chatty species, at least not with non-Quarg - i.e. they won't hail you just for the sake of saying hi.

**Comment by Lineth** [comment]
I was thinking the dialogue would only appear if you hailed them. Does it appear otherwise?

**Comment by MessyMix** [comment]
In your message log, it can. Like where it says that you've paid salaries, or that you're scanning a ship, or that you're unable to scan a ship: ship is entering hyperspace.

**Comment by Pointedstick** [comment]
Sounds like this won't be merged. Can we close it?

**Comment by Lineth** [comment]
Probably.

**Pull request closed without merging**